dora/core/common/src/main/java/alluxio/membership/StaticMembershipManager.java

```diff
@@ -100,10 +102,17 @@ private static List<WorkerInfo> parseWorkerAddresses(
             Configuration.global());
       }
       workerNetAddress.setDataPort(inetAddr.getPort());
-      workerAddrs.add(workerNetAddress);
+      // generate identities from hostnames
+      // todo(bowen): consider change the configuration file to allow specify both the
+      //  identity and the net address
+      WorkerIdentity identity = WorkerIdentity.ParserV1.INSTANCE.fromUUID(
+          UUID.nameUUIDFromBytes(workerHostname.getBytes(StandardCharsets.UTF_8)));
+      WorkerInfo workerInfo = new WorkerInfo()
+          .setIdentity(identity)
+          .setAddress(workerNetAddress);
+      workerInfos.add(workerInfo);
```

Comment:
Can worker identity configurable?

Reply:
as I noted in the todo comment above, it should be configurable by setting it in the static worker list file. Currently the file contains only worker net addresses, so the file should be changed to allow configuring both the identities and the net addresses at the same time.

Now I just generate the worker identities from the netaddresses. Do you think this is problematic?